src/sentry/search/events/datasets/spans_indexed.py

```diff
@@ -769,10 +736,18 @@ def function_converter(self) -> dict[str, SnQLFunction]:
                 ),
                 SnQLFunction(
                     "count_weighted",
-                    optional_args=[NullColumn("column")],
+                    optional_args=[
+                        with_default("span.duration", NumericColumn("column", spans=True)),
+                    ],
                     snql_aggregate=self._resolve_count_weighted,
                     default_result_type="integer",
                 ),
+                SnQLFunction(
+                    "count_unique_weighted",
```

Comment:
why are we adding this? 👀 we can't really weight count_unique right

Reply:
I see we have them for min/max below as well

Reply:
It's not actually weighted but these exist for the sake of completeness so the frontend doesn't need to think about whether or not a function can be weighted or not.